CardanoSharp.Wallet.Test/TransactionTests.cs

```diff
@@ -304,8 +304,8 @@ public void MnemonicToTransaction()
                 .WithVKeyWitnesses(new List<VKeyWitness>()
                 {
                     new VKeyWitnessBuilder()
-                        .WithVKey(paymentPub.Key)
-                        .WithSKey(paymentPrv.Key)
+                        .WithVKey(paymentPub)
+                        .WithSKey(paymentPrv)
                         .Build()
```

Comment:
You know I was just thinking about this VKeyWitnessBuilder and how FREAKIN easy it'll be to make a pool manager with this library. It makes me think though that we do need a cddl spec converter to make a pool manager easy for us

Reply:
I'm always up for a cddl converter. I just don't want to be the one to code that up lol

Reply:
Yeah me neither, and I don't think we will need it until we want to look into the prospect of a pool manager.